2022-08-03 11:01:11,160 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-08-03 11:01:11,161 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\sarka\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\instabot\bot\bot.py
2022-08-03 11:01:11,163 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-08-03 11:01:11,564 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-08-03 11:01:11,573 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-08-03 11:01:11,573 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-08-03 11:01:11,576 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
